assets/js/base/components/quantity-selector/style.scss

```diff
@@ -1,3 +1,68 @@
-.wc-block-quantity-selector__input {
-	width: 60px;
+@mixin reset-button {
+	border: 0;
+	padding: 0;
+	margin: 0;
+	background: none transparent;
+
+	&:focus {
+		outline: 2px solid #d6d9dd;
```

Comment:
This color doesn't have enough contrast, I think:

https://webaim.org/resources/contrastchecker/?fcolor=D6D9DD&bcolor=FFFFFF

Could it be replaced by something else or alternatively use the default `focus` outline?